tests/plugins/balloontoolbar/destroy.js

```diff
@@ -0,0 +1,50 @@
+/* bender-tags: balloontoolbar, 1477 */
+/* bender-ckeditor-plugins: balloontoolbar,button,basicstyles,list,richcombo,stylescombo,wysiwygarea,toolbar */
+
+( function() {
+	'use strict';
+
+	bender.editor = {
+	};
+
+	var tests = {
+		'test destroy toolbar with richcombo': function() {
```

Comment:
This test does not test anything. Array passed to `CKEDITOR.tools.array.every` is empty, therefore test always passes.